Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'decidim', DECIDIM_VERSION
 gem 'decidim-conferences', DECIDIM_VERSION
 gem 'decidim-consultations', DECIDIM_VERSION
 gem 'decidim-file_authorization_handler', git: 'https://github.com/CodiTramuntana/decidim-file_authorization_handler.git'
-gem 'decidim-initiatives', DECIDIM_VERSION
```

Comment:
We don't want to remove it, we only want to disable it during some days, so commenting it is enough